auth/server/base/src/main/java/org/wildfly/security/auth/server/RealmIdentity.java

```diff
@@ -268,6 +269,14 @@ default Attributes getAttributes() throws RealmUnavailableException {
         return null;
     }
 
+    default Set<Credential> getPublicCredentials() {
```

Comment:
Just FYI, some of the draft token authentication work can be found here:

https://github.com/wildfly-security/wildfly-elytron/pull/1541

Looks like we were using `SecurityIdentity.withPrivateCredential` there, there wasn't interaction with the realm.

Reply:
@darranl @fjuma  I added javadoc to these methods and made them return IdentityCredentials.NONE instead of null.

Btw I am not sure anymore whether the credentials from the subject are useful for us to propagate to security identity? I added it here just to preserve all the information I can from the subject. But I am not sure if these private and public credentials have its use case once they are put in the security identity.